docs/source/ide_basics.rst

```diff
@@ -0,0 +1,131 @@
+
+===================== ========== ================================== =====================================
+Sensor                Channel ID Valid Devices                      Subchannels
+===================== ========== ================================== =====================================
+Main Accelerometer    8          S-RD, S-ED, SSS, SSX               X-, Y-, Z-axis Acceleration
+--------------------- ---------- ---------------------------------- -------------------------------------
+16/200g Accelerometer 32         S-DD, SSX, SSS, SSC, S-D16, S-D200 X-, Y-, Z-axis Acceleration
+--------------------- ---------- ---------------------------------- -------------------------------------
+8/40g Accelerometer   80         S-RD, S-DD, S-ED, S-D40, S-D8      X-, Y-, Z-axis Acceleration
+--------------------- ---------- ---------------------------------- -------------------------------------
+IMU Gyroscope         47         All [1]_                           X-, Y-, Z-axis Rotation
+--------------------- ---------- ---------------------------------- -------------------------------------
+Absolute Orientation  65         All [1]_                           X-, Y-, Z-, W-axis Quaternion; Acc
+--------------------- ---------- ---------------------------------- -------------------------------------
+Relative Orientation  70         All [1]_                           X-, Y-, Z-, W-axis Quaternion
+--------------------- ---------- ---------------------------------- -------------------------------------
+MPL3115               36         All [1]_                           Pressure, Temperature [2]_
+--------------------- ---------- ---------------------------------- -------------------------------------
+MS8607                59         All [1]_                           Pressure, Temperature, Humidity [3]_
+--------------------- ---------- ---------------------------------- -------------------------------------
+SI1133                76         All [1]_                           Lux, UV
+===================== ========== ================================== =====================================
+
+.. [1] excluding early SSC/SSS/SSX models
+.. [2] 1 Hz Internal Measurements
+.. [3] 10 Hz Control Pad Measurements
+
+
+To simply use all recording data, we can iterate through each subchannel in a dataset like so::
+
+    for ch in ds.channels.values():
+        for sch in ch.subchannels:
+            print(sch)
+
+.. _eventarray_desc:
+
+EventArrays & raw data
+~~~~~~~~~~~~~~~~~~~~~~
+
+Each :class:`Channel` and :class:`Subchannel` object has a ``getSession()`` method, which returns an :class:`EventArray` object. :class:`EventArray` is a wrapper around a channel's underlying recording data that loads data on demand from the source file. You can index an :class:`EventArray` (e.g., ``eventarray[i]`` for some index ``i``) to get a numpy ``ndarray`` of data. Data is organized in an n-dimensional array.
```

Comment:
`SubChannel` has a capital "C"

Reply:
addressed in f34863e2684b9ee9eddd485954b5e896cdc7e481